### PR TITLE
Restructure logs

### PR DESCRIPTION
**Pull request by mateimicu**
Log more informations for debugging purposes.

Signed-off-by: Micu Matei-Marius mmicu@cloudbasesolutions.com

### CODE REVIEW

File: argus/action_manager/windows.py [hunk 1]
```diff
@@ -239,10 +238,11 @@ def remove(self, path):
     def rmdir(self, path):
         """Remove a directory."""
         if not self.exists(path):
-            raise exceptions.ArgusCLIError("Invalid Path.")
+            raise exceptions.ArgusCLIError("Invalid Path '{}'.".format(path))
 
         if not self.is_file(path):
```
Comment (stefan-caraiman) [inline argus/action_manager/windows.py]: So if it is not a file, the path is not a directory? Correct me if I'm mistaken, but should't it be, `The path {} is not a file` since you are checking if the path is not a file, or am I missing something?

File: argus/action_manager/windows.py [hunk 2]
```diff
@@ -239,10 +238,11 @@ def remove(self, path):
...
-            raise exceptions.ArgusCLIError("The path is not a directory.")
```
Comment (stefan-caraiman) [inline argus/action_manager/windows.py:245]: So if it is not a file, the path is not a directory? Correct me if I'm mistaken, but should't it be, `The path {} is not a file` since you are checking if the path is not a file, or am I missing something?

File: argus/action_manager/windows.py [hunk 3]
```diff
@@ -306,7 +306,7 @@ def mkdir(self, path):
         """
         if self.exists(path):
             raise exceptions.ArgusCLIError(
-                "Cannot create directory ‘{}’ The path already exists".format(
+                "Cannot create directory '{}'. The path already exists".format(
```
Comment (stefan-caraiman) [inline argus/action_manager/windows.py]: I think it would be more appropriate to log `Cannot create directory {} . It already exists.`

File: argus/action_manager/windows.py [hunk 4]
```diff
@@ -318,13 +318,15 @@ def mkfile(self, path):
             Remote path where the new file should be created.
         """
         if self.is_file(path):
-            LOG.warning("File already exists."
-                        " LastWriteTime and LastAccessTime will be updated.")
+            LOG.warning("File already exists '{}'. LastWriteTime and"
```
Comment (stefan-caraiman) [inline argus/action_manager/windows.py]: I think it would look better to log it like this `File '{}' already exists. LastWrite....`

File: argus/action_manager/windows.py [hunk 5]
```diff
@@ -318,13 +318,15 @@ def mkfile(self, path):
...
+                        " LastAccessTime will be updated.").format(path)
             self._client.run_command_with_retry(
                 "echo $null >> '{}'".format(path),
                 command_type=util.POWERSHELL)
         elif self.is_dir(path):
-            raise exceptions.ArgusCLIError("Path leads to a directory.")
+            raise exceptions.ArgusCLIError(
+                "Path '{}' leads to a"
+                                           " directory.".format(path))
```
Comment (stefan-caraiman) [inline argus/action_manager/windows.py]: Is this white space necessary?

File: argus/client/windows.py [hunk 6]
```diff
@@ -161,11 +161,9 @@ def write_file(self, data, remote_destination):
         The remote destination is the file name where the content
         of filepath will be written.
         """
-        LOG.debug("Write data in file %s", remote_destination)
-
         # TODO(mmicu): This powershell dance is a little complicated,
-        # find a simpler way to send a file over a remote server,
-        # without relying on OpenStack infra.
+        #              find a simpler way to send a file over a remote server,
```
Comment (stefan-caraiman) [inline argus/client/windows.py]: Ditto

File: argus/config_generator/windows/base.py [hunk 7]
```diff
@@ -83,6 +85,8 @@ def apply_config(self, path):
         self.conf.write(buff)
         buff.seek(0)
         data = buff.read()
+
+        LOG.debug("Write data in file %s", file_path)
```
Comment (stefan-caraiman) [inline argus/config_generator/windows/base.py]: `Writing data in file %s`

File: argus/recipes/cloud/windows.py [hunk 8]
```diff
@@ -133,12 +133,12 @@ def replace_code(self):
...
         LOG.info("Replacing cloudbaseinit's code...")
 
-        LOG.info("Getting cloudbase-init location...")
+        LOG.debug("Getting cloudbase-init location...")
         # Get cb-init python location.
         python_dir = introspection.get_python_dir(self._execute)
 
         # Remove everything from the cloudbaseinit installation.
-        LOG.info("Removing recursively cloudbaseinit...")
+        LOG.debug("Removing recursively cloudbaseinit...")
```
Comment (stefan-caraiman) [inline argus/recipes/cloud/windows.py]: Please rephrase it to `Recursively removing ...`

File: argus/recipes/cloud/windows.py [hunk 9]
```diff
@@ -427,7 +429,7 @@ class CloudbaseinitLocalScriptsRecipe(CloudbaseinitRecipe):
 
     def pre_sysprep(self):
         super(CloudbaseinitLocalScriptsRecipe, self).pre_sysprep()
-        LOG.info("Download reboot-required local script.")
+        LOG.debug("Download reboot-required local script.")
```
Comment (stefan-caraiman) [inline argus/recipes/cloud/windows.py]: *Downloading

File: argus/action_manager/windows.py (hunk 2 above)
Comment (mateimicu) [reply argus/action_manager/windows.py:245]: You are right, this is a known bug that is fixed in this #38 PR.

File: argus/action_manager/windows.py [hunk 10]
```diff
@@ -126,8 +126,6 @@ def install_cbinit(self):
             build=self._conf.argus.build,
```
Comment (ader1990) [inline argus/action_manager/windows.py:126]: typo in commit message

File: argus/client/windows.py [hunk 11]
```diff
@@ -140,9 +140,9 @@ def copy_file(self, filepath, remote_destination):
         of filepath will be written.
         """
 
-        # TODO(cpopa): This powershell dance is a little complicated,
-        # find a simpler way to send a file over a remote server,
-        # without relying on OpenStack infra.
+        # TODO(mmicu): This powershell dance is a little complicated,
```
Comment (ader1990) [inline argus/client/windows.py]: don't change the name of the one who wrote the note

**Comment by stefan-caraiman** [comment]
`enchant` isn't really the right term for this commit message or context, it would be better to use `improve`

File: argus/recipes/cloud/windows.py [hunk 12]
```diff
@@ -324,6 +323,9 @@ def prepare_cbinit_config(self, service_type):
         super(CloudbaseinitMockServiceRecipe, self).prepare_cbinit_config(service_type)
         LOG.info("Inject guest IP for mocked service access.")
 
+
+        # TODO(mmicu): The service type is specific to each scenario,
+        #              find a way to pass it.
```
Comment (ader1990) [inline argus/recipes/cloud/windows.py]: for the todo indentation, why don't we keep it clean and in sync with the openstack one?
https://github.com/openstack/nova/blob/bc5035343d366a18cae587f92ecb4e871aba974a/nova/scheduler/filters/retry_filter.py#L37

File: etc/argus.conf.sample [hunk 13]
```diff
@@ -1,4 +1,4 @@
-# TODO(cpoieana): fix configuration sample according to the new Argus
+# TODO(mmicu): fix configuration sample according to the new Argus
```
Comment (ader1990) [inline etc/argus.conf.sample]: same as above

**Comment by alexcoman** [comment]
LGTM;
